src/components/JdsSearch/Search.stories.js

```diff
@@ -0,0 +1,64 @@
+import JdsSearch from './Search.vue'
+import { default as storybookMixin } from '../../utils/storybook'
+
+export default {
+  component: JdsSearch,
+  title: 'Components/Search',
+}
+
+const Template = (args, context) => {
+  return {
+    name: 'JdsSearchStories',
+    components: { JdsSearch },
+    props: Object.keys(context.argTypes),
+    mixins: [storybookMixin(args, context)],
+    template: `
+    <jds-search 
+      v-bind="$props"
+      v-on="events"
+    />
+    `,
+  }
+}
+
+const defaultArgs = {
+  icon: true,
+  button: true,
+  placeholder: 'My Placeholder'
+};
+
+export const Default = Template.bind({})
+Default.args = {
+  value: '',
+  ...defaultArgs,
+}
+
+export const WithSmallButton = Template.bind({})
+WithSmallButton.args = {
+  value: '',
+  ...defaultArgs,
+  button: true,
+  small: true
+}
+
+export const WithoutIcon = Template.bind({})
+WithoutIcon.args = {
+  value: '',
+  ...defaultArgs,
+  icon: false
+}
+
+export const WithoutButton = Template.bind({})
+WithoutButton.args = {
+  value: '',
+  ...defaultArgs,
+  button: false
+}
+
+export const WithoutButtonAndIcon = Template.bind({})
+WithoutButtonAndIcon.args = {
+  value: '',
+  ...defaultArgs,
+  button: false,
+  icon: false
+}
```

Comment:
**minor**
better you can give new line end of file

Reply:
Thanks, done [here](https://github.com/jabardigitalservice/jds-design-system-vue-lib/pull/119/commits/b8af091073a0867ffcb1d05bbb6c17309caa76b0)